packages/metal-component/src/Component.js

```diff
@@ -439,8 +439,8 @@ class Component extends EventEmitter {
 		}
 		const instance = new Ctor(config, false);
 
-		if (window.__METAL_DEV_TOOLS_HOOK__) {
-			window.__METAL_DEV_TOOLS_HOOK__(instance);
+		if (window.__METAL_DEV_TOOLS_HOOK__ && window.__METAL_DEV_TOOLS_HOOK__.addRoot) {
+			window.__METAL_DEV_TOOLS_HOOK__.addRoot(instance);
```

Comment:
Hey @bryceosterhaus, now that it's clear that we don't want to keep backwards compatibility, then this brings up my second concern, which is that every change in `metal-devtools` might need a corresponding change in devtools since this couples `metal.js` to a specific devtools implementation (now the `addRoot` method).

Are we doing this because you expect more hooks to be needed? I'm asking because I can see different ways to go about this.

**Only the render hook is needed**
Then we could keep this as is and simply refactor the devtools, so we keep `METAL_DEV_TOOLS_HOOK` as the entry point which takes care of calling any additional methods. This keeps `metal.js` agnostic of any `devtools` implementation.

**More than one hook is needed**
In this case, makes sense that a simple entry point won't be enough in the long run, so we could either do:

- Pass additional standard context information for the entry point to dispatch the actions
```javascript
window.__METAL_DEV_TOOLS_HOOK_(instance, {clazz: Component, method: 'render'})
```

- Stablish a proper convention after the method that is being hooked
```javascript
window.__METAL_DEV_TOOLS_HOOK_.render(instance);
// or
window.__METAL_DEV_TOOLS_HOOK_['Component::render'](instance);
```

Does any of that makes sense?